### DISCUSSION TITLE
Github search not list all my repositories

### DISCUSSION THREAD

**Discussion by Xarkam** [community/community#22231] [New to GitHub]
<p>Hello,</p>
<p>I don’t understand why when searching for my organization name only 2 repositories are given as a result.</p>
<p>Do I have something in my other repositories that is blocking indexing by google?</p>
<p>(My organisation is softinux)<br>
My repositories<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/4/742e8180f10979c102e1a5ad1f69e605f64e6c6a.png" data-download-href="/uploads/short-url/gzN1xTA46unzY7Vz2mDxZYW3z0m.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/331816/181069649-8711e979-f4dc-4167-9c7f-25fd814dcdb3.png" alt="image" data-base62-sha1="gzN1xTA46unzY7Vz2mDxZYW3z0m" width="560" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/4/742e8180f10979c102e1a5ad1f69e605f64e6c6a_2_560x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/4/742e8180f10979c102e1a5ad1f69e605f64e6c6a_2_840x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/4/742e8180f10979c102e1a5ad1f69e605f64e6c6a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/4/742e8180f10979c102e1a5ad1f69e605f64e6c6a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">991×884 85.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
Search result<br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/a/aa4eea7d88c47ce47d4ae86cc519578270024ef0.png" data-download-href="/uploads/short-url/oiCfieCayo1BxKk1OJ3wUjXjJqU.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/331816/181069651-989eccd0-a70e-44cc-b5f8-87a60bbea91b.png" alt="image" data-base62-sha1="oiCfieCayo1BxKk1OJ3wUjXjJqU" width="690" height="395" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aa4eea7d88c47ce47d4ae86cc519578270024ef0_2_690x395.png, https://aws1.discourse-cdn.com/github/original/3X/a/a/aa4eea7d88c47ce47d4ae86cc519578270024ef0.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/a/aa4eea7d88c47ce47d4ae86cc519578270024ef0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/a/aa4eea7d88c47ce47d4ae86cc519578270024ef0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1025×588 40.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/xarkam">@xarkam</a> :wave:</p>
<p>Love this question! Your results only yield a couple of your repositories because they have your org name listed in its description. Our general search looks for keywords in the description of a repository.</p>
<p>The better search here would be to use:</p>
<pre><code class="lang-auto">org:SOFTINUX
</code></pre>
<p>This   search will return all repository matching that org name.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="15" height="15">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/searching-for-repositories#search-within-a-users-or-organizations-repositories" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="20" height="20">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/searching-for-repositories#search-within-a-users-or-organizations-repositories" target="_blank" rel="noopener">Searching for repositories - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Xarkam** [comment]
<p>Thank you, your answer is clear and precise.<br>
I modify the “about” of the repositories and the search gives the result I want.</p>
<p>Thanks again.</p>
